src/diffusers/schedulers/scheduling_k_dpm_2_ancestral_discrete.py

```diff
@@ -305,6 +315,44 @@ def sigma_to_t(self, sigma):
         t = t.view(sigma.shape)
         return t
 
+    # Copied from diffusers.schedulers.scheduling_euler_discrete.EulerDiscreteScheduler._sigma_to_t
+    def _sigma_to_t(self, sigma, log_sigmas):
```

Comment:
@patrickvonplaten 
We have a `sigma_to_t` function that does the same math as this one but is calculated with pytorch tensors (vs np here). Should we update all the calculations of sigmas in the np array so that we can just use `_sigma_to_t`?

Reply:
I think it doesn't really matter whether we use PyTorch or numpy here. Ideally we have the same `sigma_to_t` (or `_sigma_to_t` function across all schedulers though

Reply:
I replaced `sigma_to_t` with `_sigma_to_t` :)

Reply:
works for me!